src/PubSubClient.cpp

```diff
@@ -12,93 +12,125 @@ PubSubClient::PubSubClient() {
     this->_client = NULL;
     this->stream = NULL;
     setCallback(NULL);
+    this->buffer_size = MQTT_MAX_PACKET_SIZE;
```

Comment:
Woudn't it be better to move the buffer allocation to a support function, rather than duplicate the same code in all constructors?

Reply:
i agree in principal but
i was trying to match the existing code style. (ie, explicitly calling everything in each constructor.)

@knolleary any opinion?